src/utils/network.ts

```diff
@@ -0,0 +1,24 @@
+import { createServer } from 'net';
```

Comment:
Please do not forget to add the copyright banners when creating new files.
```suggestion
/*
 * Copyright © 2023 Lisk Foundation
 *
 * See the LICENSE file at the top-level directory of this distribution
 * for licensing information.
 *
 * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
 * no part of this software, including this file, may be copied, modified,
 * propagated, or distributed except according to the terms contained in the
 * LICENSE file.
 *
 * Removal or modification of this copyright notice is prohibited.
 */
 import { createServer } from 'net';
```

Reply:
Applied.